README.md

```diff
@@ -38,6 +38,25 @@ When there's a `.pbxproj` file with Git conflicts, and a 3-way merge is possible
 
 And see the magic happen! :sparkles:
 
+### Git merge driver
+
+To use Kintsugi as a Git merge driver, follow these steps:
+
+- Add it as driver to Git config file by running the following:
+```sh
+git config merge.kintsugi.name "Kintsugi driver" # Or any other name that your prefer
+git config merge.kintsugi.driver "kintsugi driver %O %A %B %P"
+```
+
+`kintsugi` should be in your `PATH`.
+Run `git config` with `--global` to add this to the global config file.
+
+- Set the the merge driver of the repository to use Kintsugi for `.pbxproj` files, by adding the following line to the `.gitattributes` file of the repository:
+
+`*.pbxproj merge=kintsugi`
+
+You can set this globally by putting it in `$HOME/.config/git/attributes`.
```

Comment:
be more accurate about the path of git attributes (you can still keep an example) and link to the official git doc page.